pulp_container/app/models.py

```diff
@@ -199,6 +199,13 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
         unique_together = (("name", "tagged_manifest"),)
 
+    @classmethod
+    def natural_key_fields(cls):
+        """
+        Returns a tuple of the natural key fields which usually equates to unique_together fields
+        """
+        return ("name", "tagged_manifest_id")
```

Comment:
Same here

Reply:
"natural_key_fields" is a pulp invention. Is there a way we could be more clever on the pulpcore side?
Just thinking. Not blocking here.

Reply:
We probably could, by doing something like [this](https://github.com/rsinger86/django-lifecycle/pull/64/commits/eed6c922b8e12a6397b7c037723b366bcc88371c#diff-ccd0462c3332b1460bdeefa901260186d8b9de888e6acd16c6d63a0fbbbe2c27R161-R163). 

This feels OK as a short-term solution though.  This is a pretty rare case.

Reply:
Looks like this could work. But yes, we should cache the result.

https://github.com/pulp/pulpcore/pull/1625